templates/taxi/car_list.html

```diff
@@ -1,16 +1,28 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a  class="btn btn-primary" style="float: right" href="{% url 'taxi:car-create' %}" role="button">Create</a></h1>
```

Comment:
Ensure there is consistent spacing around the attributes in the `<a>` tag. There is an extra space before the `class` attribute. Consider removing this extra space for consistency and to adhere to HTML best practices.